project/libs/blog-module/src/dto/create-post.dto.ts

```diff
@@ -0,0 +1,9 @@
+export class CreatePostDto {
+	public typeUuid: string;
+	public userUuid: string;
```

Comment:
Лучше назвать userId. Если вдруг иметь  id в виде числа не придётся менять названия 
```suggestion
	public userId: string;
```